ui/ui-frontend/projects/collect/src/app/collect/archive-search-collect/archive-preview/collect-object-group-details-tab/collect-object-group-details-tab.component.spec.ts

```diff
@@ -63,6 +64,21 @@ describe('CollectObjectGroupDetailsTabComponent', () => {
     'getObjectGroupDetailsById',
   ]);
 
+  const tenantSelectionServiceSpy = jasmine.createSpyObj('TenantSelectionService', {
+    getSelectedTenant: {
+      name: 'tenantName',
+      identifier: 2,
+      ownerId: 'owner',
+      customerId: 'customer',
+      enabled: true,
+      proof: false,
+      readonly: true,
+      ingestContractHoldingIdentifier: 'string',
+      itemIngestContractIdentifier: 'string',
+      accessContractHoldingIdentifier: 'string',
+      accessContractLogbookIdentifier: 'string',
+    },
+  });
```

Comment:
Ce n'est pas suffisant ?
```suggestion
  const tenantSelectionServiceSpy = createSpyObj('TenantSelectionService', ['getSelectedTenant']);
```

Reply:
Il s'attend au champs identifier

Reply:
Ma remarque est qu'on a 2 fois l'objet: une fois dans le spy, une autre fois plus bas. Je voulais éviter la duplication.

Reply:
je vois, corrigé